htdocs/core/class/html.formticket.class.php

```diff
@@ -1394,7 +1394,7 @@ public function showMessageForm($width = '40%')
 
 		$result = $formmail->fetchAllEMailTemplate($this->param["models"], $user, $outputlangs);
 		if ($result < 0) {
-			setEventMessages($this->error, $this->errors, 'errors');
```

Comment:
The setEventMessages replace the old setEventMessage. So you must keep the version with the s.
Instead to fix the warning, you can just declare the property ->errors = array() into the class FormTicket.

Reply:
okey perfect